TUTORIAL.md

```diff
@@ -9,20 +9,17 @@ In order to complete this tutorial successfully, there are a few things that you
 You can check to see if you have pip installed by opening up Terminal and typing the following command: pip --version
 If you see a version number, pip is installed, and you're ready to proceed. If not, instructions for downloading the latest version of pip can be found      here: https://pip.pypa.io/en/stable/cli/pip_download/
 
+A. After ensuring that pip is installed, run the following command in Terminal to install Spotipy: pip install spotipy --upgrade
+
+A. After ensuring that pip is installed, run the following command in Terminal to install Spotipy: pip install spotipy --upgrade
+
```

Comment:
Not sure how this happened, but it seems that this line is duplicated.
```suggestion
A. After ensuring that pip is installed, run the following command in Terminal to install Spotipy: pip install spotipy --upgrade

```

Reply:
This is not your fault, but I have to mention it somewhere. Why do we check if pip is installed and then AFTER we check if Python is installed? I think we could change the order as well.

Reply:
Noted about the duplicate line. I have removed this.

Good callout about the order of pip and Python installation instructions. I have changed the order of these.

It looks like my pull request is failing due to a couple of failed tests in the non_user_endpoints/test.py file. Would you suggest I comment out those tests or do something else? My local repo should be up-to-date with the remote repo, so I'm not quite sure why those tests are failing unless they're also failing for everyone else.